import-service/src/functions/import-file-parser/handler.ts

```diff
@@ -0,0 +1,66 @@
+import {
+  CopyObjectCommand,
+  DeleteObjectCommand,
+  GetObjectCommand,
+  S3Client,
+} from "@aws-sdk/client-s3";
+import { formatJSONErrorResponse, formatJSONResponse } from "@libs/api-gateway";
+import { middyfy } from "@libs/lambda";
+import { Handler, S3Event } from "aws-lambda";
+import { s3ClientConfig } from "src/config/s3-client-config";
+import { PARSED_FILES_FOLDER, UPLOADS_FOLDER } from "src/config/constants";
+
+const csv = require("csv-parser");
+
+const importFileParser: Handler<S3Event> = async ({ Records }) => {
+  try {
+    const record = Records[0];
+
+    if (!record) {
+      return formatJSONErrorResponse("Uploaded file not found");
+    }
+
+    const bucket = record.s3.bucket.name;
+    const key = record.s3.object.key;
+
+    const commandConfig = {
+      Bucket: bucket,
+      Key: key,
+    };
+
+    const client = new S3Client(s3ClientConfig);
```

Comment:
Create `clients` of any sort outside lambda always. Otherwise your lambda will spend more run time thus more money per each scaled function.

https://docs.aws.amazon.com/lambda/latest/dg/best-practices.html (the very first point)